compiler/include/symbol.h

```diff
@@ -353,10 +353,18 @@ class FnSymbol : public Symbol {
   int codegenUniqueNum;
   const char *doc;
 
+  // The following fields are used only when hasFlag(FLAG_PARTIAL_COPY)
+  // to pass information from partialCopy() to finalizeCopy().
+  // This localized use suggests that they should be moved to a hashtable.
```

Comment:
Please remove the last line.  I don't think that's the right path and
wouldn't want to lead someone astray.